_archive/mv2/extensions/news/README

```diff
@@ -0,0 +1,7 @@
+# News
+
+This sample demonstrates a simple popup panel that access a remote URL. In this case, we are the Google News RSS feed, and building a UI to display that information.
```

Comment:
```suggestion
This sample demonstrates a popup panel that accesses a remote URL (in this case, a Google News RSS feed) and builds a UI to display that information.
```

Reply:
The corporate style guide likes us to avoid words like 'simple' and 'easy'. Removing it also applies a writing best practice to prefer saying things with fewer words when possible.